src/Core/Framework/Migration/Command/MigrationCommand.php

```diff
@@ -148,10 +145,10 @@ private function runMigrationForIdentifier(InputInterface $input, string $identi
 
         try {
             $collection = $this->collectMigrations($input, $identifier);
-        } catch (UnknownMigrationSourceException $e) {
+        } catch (UnknownMigrationSourceException) {
             $this->io->note(\sprintf('No collection found for identifier: "%s", continuing', $identifier));
 
-            return self::SUCCESS;
+            return 0;
```

Comment:
Why don't use the constant?

Reply:
look at the other return points, it is returning a count, not a state :grimacing:

Reply:
ah :see_no_evil: